client/src/i18n/en/tree.json

```diff
@@ -147,6 +147,7 @@
     "STOCK_ENTRY_REPORT":"[Stock] Entries Report",
     "STOCK_EXIT_REPORT" : "[Stock] Exit Report",
     "STOCK_INVENTORY" : "Items in Stock",
+    "STOCK_EXPIRATION_REPORT": "[Stock] expiration report",
```

Comment:
I suggest capitalizing Expiration and Report here to match the other entries.

Reply:
okay